sql/log.cc

```diff
@@ -6390,6 +6357,46 @@ bool stmt_has_updated_non_trans_table(const THD* thd)
   return (thd->transaction->stmt.modified_non_trans_table);
 }
 
+/**
+  Updates the skip status of the binlog cache based on the current event.
+
+  If the event has the LOG_EVENT_SKIP_REPLICATION_F flag set,
+  Then there are two options either this is is part of a
+  query that affects a single table/db or it would be part 
+  of a partial filter.
+
+  If it's the first case then all events in an event group should be
+  filtered , this is done by updating the cache's event group filter
+  flag. Otherwise then it's part of the partial filter which should have 
```

Comment:
Something is off in this description. There's some gap in logic between "part of a query that affects a single table/db" and "all events in an event group should be filtered".